resources/aws/environment.go

```diff
@@ -15,17 +15,18 @@ const (
 	awsRegionParamName = "region"
 
 	// AWS Infra
-	DDInfraDefaultVPCIDParamName           = "aws/defaultVPCID"
-	DDInfraDefaultSubnetsParamName         = "aws/defaultSubnets"
-	DDInfraDefaultSecurityGroupsParamName  = "aws/defaultSecurityGroups"
-	DDInfraDefaultInstanceTypeParamName    = "aws/defaultInstanceType"
-	DDInfraDefaultARMInstanceTypeParamName = "aws/defaultARMInstanceType"
-	DDInfraDefaultKeyPairParamName         = "aws/defaultKeyPairName"
-	DDinfraDefaultPublicKeyPath            = "aws/defaultPublicKeyPath"
-	DDInfraDefaultPrivateKeyPath           = "aws/defaultPrivateKeyPath"
-	DDInfraDefaultPrivateKeyPassword       = "aws/defaultPrivateKeyPassword"
-	DDInfraDefaultInstanceStorageSize      = "aws/defaultInstanceStorageSize"
-	DDInfraDefaultShutdownBehavior         = "aws/defaultShutdownBehavior"
+	DDInfraDefaultVPCIDParamName                 = "aws/defaultVPCID"
+	DDInfraDefaultSubnetsParamName               = "aws/defaultSubnets"
+	DDInfraDefaultSecurityGroupsParamName        = "aws/defaultSecurityGroups"
+	DDInfraDefaultInstanceTypeParamName          = "aws/defaultInstanceType"
+	DDInfraDefaultDockerInstanceProfileParamName = "aws/defaultInstanceProfile"
```

Comment:
I don't think it should be specific to Docker. The need to specify an instanceProfile can be generic.